### PR TITLE
docs: trans of core

### PR DESCRIPTION
**Pull request by Qianmoxsn**

### CODE REVIEW

**Comment by Qianmoxsn** [comment]
All docs in core have been translated, please check the work
@lslzl3000 @Noah133552

File: docs/guide/core/object.md [hunk 1]
```diff
@@ -1,67 +1,69 @@
...
 }
 ```
 
-## 获取节点的其它组件
-通过 `Object3D` 自带的 [getComponent](/api/classes/Object3D#getComponent) 方法，可以方便的获取节点上的组件。
+## Get Other Components of Node
+Using the built-in [getComponent](/api/classes/Object3D#getComponent) method of `Object3D`, you can easily get the components on the node.
 ```ts
-//自定义一个组件，效果是变更节点上的另一个灯光组件，改变灯光的颜色
+//Custom component, to change another light component on the node, change the color of the light
```
Comment (Noah133552) [inline docs/guide/core/object.md:66]: Customize a component.

Double-check the purpose of this coding block: is it to change another light component's color on the node

Comment (Qianmoxsn) [reply docs/guide/core/object.md:66]: I'm not sure, but the chinese document says so

File: docs/guide/core/object.md [hunk 2]
```diff
@@ -1,67 +1,69 @@
...
 }
 ```
 
-## 获取节点的其它组件
-通过 `Object3D` 自带的 [getComponent](/api/classes/Object3D#getComponent) 方法，可以方便的获取节点上的组件。
+## Get Other Components of Node
+Using the built-in [getComponent](/api/classes/Object3D#getComponent) method of `Object3D`, you can easily get the components on the node.
 ```ts
-//自定义一个组件，效果是变更节点上的另一个灯光组件，改变灯光的颜色
+//Custom a component, to change another light component on the node, change the color of the light
```
Comment (Noah133552) [inline docs/guide/core/object.md:66]: Customize a component

**Review by Noah133552 — APPROVED** [review]

**Pull request merged**